IronRoadForChildren/Features/Program/Day/ProgramItemView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ProgramItemView.swift
+//  IronRoadForChildren
+//
+//  Created by Robert Zavaczki on 18.04.23.
+//
+
+import SwiftUI
+
+struct ProgramItemView: View {
+	var body: some View {
+		Rectangle()
+			.cornerRadius(10)
+			.frame(height: 80)
+			.foregroundColor(.pink.opacity(0.03))
+			.overlay(
+				RoundedRectangle(cornerRadius: 20)
+					.strokeBorder(Color.gray, lineWidth: 1)
+			)
+			.overlay(
+				HStack {
+					Image(systemName: "person.fill")
+						.resizable()
+						.frame(height: 80)
+						.aspectRatio(1, contentMode: .fit)
+					VStack(alignment: .leading, spacing: 5) {
+						Text("THE BAND THAT HAS A VERY LONG NAME")
+							.fontWeight(.semibold)
+							.lineSpacing(5)
+							.font(.system(size: 16))
```

Comment:
The color looks not black in the screenshot and I would recommend finding a system default forn which we could use (i.e. `.subheading`)